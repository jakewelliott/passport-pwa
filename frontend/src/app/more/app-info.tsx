import { PassportHeader } from '../../components/common/passport-header';
import { ContactUs } from '../../components/tabs/more/contact-us';
import { DownloadParkMaps } from '../../components/tabs/more/download-park-maps';
import { Reservations } from '../../components/tabs/more/reservations';

export const AppInfo = () => {
  return (
    <div className='max-w-[500px] mx-auto'>
      <div className='my-3.5'>
        <PassportHeader />
      </div>
      <ContactUs />
      <DownloadParkMaps />
      <Reservations />
    </div>
  );
};
