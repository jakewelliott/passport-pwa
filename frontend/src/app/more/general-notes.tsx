import { NotesMiniTab } from '../locations/components/notes-minitab';

export const EditGeneralNotes = () => {
	return (
		<div className='container mx-auto'>
			<div className='flex h-[calc(100vh-1.5rem-100px)] flex-col'>
				<NotesMiniTab parkId={0} />
			</div>
		</div>
	);
};
