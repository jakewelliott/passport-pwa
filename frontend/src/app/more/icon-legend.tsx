import { LegendSections } from '@/app/more/components/legend-sections';
import { dbg } from '@/lib/debug';

export const IconLegend = () => {
    dbg('RENDER', '/more/icon-legend');
    return (
        <div className='flex flex-col gap-4'>
            <h2 className='text-center'>Legend</h2>
            <LegendSections />
            <p className='min-w-[200px]'>
                <span className='text-icon-amenities'>Kids TRACK Trail: </span>Kids in Parks is a group that creates
                outdoor adventures called TRACK trails for families all over the country. Each one has fun things to
                learn about nature as you walk. You can even earn prizes by keeping track of your adventures! Check out
                kidsinparks.com to find out more.
            </p>
        </div>
    );
};
