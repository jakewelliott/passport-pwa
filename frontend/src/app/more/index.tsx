import RoundedButton from "../../components/common/rounded-button";

export default function More() {
	return (
		<>
			<h1>More</h1>
			<RoundedButton title="TEST" color="system_black" />
		</>
	);
}
