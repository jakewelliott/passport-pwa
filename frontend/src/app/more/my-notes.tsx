import { useState, useEffect } from 'react';
import { useNavigate } from 'react-router-dom';
import ListRow from '../../components/common/list-row';
import { a11yOnClick } from '@/lib/a11y';

export const MyNotes = () => {
  const [generalNotes, setGeneralNotes] = useState('');
  const navigate = useNavigate();
  const parks = [
    {
      name: 'Sample Data',
      abbreviation: 'ABCD',
      address: [
        {
          name: 'Main Address:',
          addressLineOne: '1234 Main St',
          city: 'Raleigh',
          state: 'NC',
          zip: '27606',
        },
        {
          name: 'Secondary Address:',
          addressLineOne: '1234 Main St',
          city: 'Raleigh',
          state: 'NC',
          zip: '27606',
        },
      ],
      coordinates: '35.2023, -78.9761',
      phone: '(555) 555-5555',
      email: 'email@ncparks.gov',
      website: 'ncparks.gov',
      stamp: {
        time: '1/1/25',
        method: 'manual',
      },
      bucketList: {
        text: 'random bucket list item',
        status: false,
      },
      established: '2003',
      landmark: 'My House',
      youCanFind: 'My items',
      trails: 'The driveway',
      parkIcons: ['Paddling-Red.svg', 'RVCamping-Green.svg', 'Playground-Blue.svg'],
      parkPhotos: [
        { url: './photos/CABE.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
      ],
      parkNotes: '',
    },
    {
      name: 'Sample Data',
      abbreviation: 'ABCD',
      address: [
        {
          name: 'Main Address:',
          addressLineOne: '1234 Main St',
          city: 'Raleigh',
          state: 'NC',
          zip: '27606',
        },
        {
          name: 'Secondary Address:',
          addressLineOne: '1234 Main St',
          city: 'Raleigh',
          state: 'NC',
          zip: '27606',
        },
      ],
      coordinates: '35.2023, -78.9761',
      phone: '(555) 555-5555',
      email: 'email@ncparks.gov',
      website: 'ncparks.gov',
      stamp: {
        time: '1/1/25',
        method: 'manual',
      },
      bucketList: {
        text: 'random bucket list item',
        status: false,
      },
      established: '2003',
      landmark: 'My House',
      youCanFind: 'My items',
      trails: 'The driveway',
      parkIcons: ['Paddling-Red.svg', 'RVCamping-Green.svg', 'Playground-Blue.svg'],
      parkPhotos: [
        { url: './photos/CABE.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
      ],
      parkNotes: '',
    },
    {
      name: 'Sample Data',
      abbreviation: 'ABCD',
      address: [
        {
          name: 'Main Address:',
          addressLineOne: '1234 Main St',
          city: 'Raleigh',
          state: 'NC',
          zip: '27606',
        },
        {
          name: 'Secondary Address:',
          addressLineOne: '1234 Main St',
          city: 'Raleigh',
          state: 'NC',
          zip: '27606',
        },
      ],
      coordinates: '35.2023, -78.9761',
      phone: '(555) 555-5555',
      email: 'email@ncparks.gov',
      website: 'ncparks.gov',
      stamp: {
        time: '1/1/25',
        method: 'manual',
      },
      bucketList: {
        text: 'random bucket list item',
        status: false,
      },
      established: '2003',
      landmark: 'My House',
      youCanFind: 'My items',
      trails: 'The driveway',
      parkIcons: ['Paddling-Red.svg', 'RVCamping-Green.svg', 'Playground-Blue.svg'],
      parkPhotos: [
        { url: './photos/CABE.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
      ],
      parkNotes: 'heyyo',
    },
    {
      name: 'Sample Data',
      abbreviation: 'ABCD',
      address: [
        {
          name: 'Main Address:',
          addressLineOne: '1234 Main St',
          city: 'Raleigh',
          state: 'NC',
          zip: '27606',
        },
        {
          name: 'Secondary Address:',
          addressLineOne: '1234 Main St',
          city: 'Raleigh',
          state: 'NC',
          zip: '27606',
        },
      ],
      coordinates: '35.2023, -78.9761',
      phone: '(555) 555-5555',
      email: 'email@ncparks.gov',
      website: 'ncparks.gov',
      stamp: {
        time: '1/1/25',
        method: 'manual',
      },
      bucketList: {
        text: 'random bucket list item',
        status: false,
      },
      established: '2003',
      landmark: 'My House',
      youCanFind: 'My items',
      trails: 'The driveway',
      parkIcons: ['Paddling-Red.svg', 'RVCamping-Green.svg', 'Playground-Blue.svg'],
      parkPhotos: [
        { url: './photos/CABE.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
        { url: './photos/CACR.jpg' },
      ],
      parkNotes:
        ' this is some super lajnfiawlrbjhvkjg hcfjtgfkgvkjhvkyufvvvvvvvv vvvvvvvvvvvvvvvvvvvvvv vvvvvvvvvvvvvvvvvvvvvvvvvvvvvv h hvhvhvhvhvhvhvhvhvhvhvhvhvhvhvhvhv hvhvhvhvhvhvhvhvhvhvhvhvhvhvhvhvhvhvh vhvhvhvhvhvhvhvhvhvhvhvhvhvhv hvhvhvhvhvhvhvhvhvhvhvhhvhv',
    },
  ];
  const parksWithNotes = parks.filter((park) => park.parkNotes && park.parkNotes.trim() !== '');

  useEffect(() => {
    const savedNotes = localStorage.getItem('generalNotes');
    if (savedNotes) {
      setGeneralNotes(savedNotes);
    }
  }, []);

  return (
    <div className='container mx-auto px-4 py-4'>
      <div className='space-y-4'>
        <div {...a11yOnClick(() => navigate('/more/my-notes/general-notes'))} className='cursor-pointer'>
          <ListRow>
            <div className='flex flex-col gap-1'>
              <h3>General Notes</h3>
              <p className='mb-2 text-gray-500 text-sm'>Last updated: Not available</p>
              <p
                className='overflow-wrap-anywhere line-clamp-3 max-w-full hyphens-auto break-words'
                style={{ hyphens: 'auto' }}
              >
                {generalNotes || 'No general notes yet'}
              </p>
            </div>
          </ListRow>
        </div>

        {/* Park Notes section */}
        {parksWithNotes.length === 0 ? (
          <p className='text-gray-600'>No park notes found.</p>
        ) : (
          parksWithNotes.map((park, index) => (
            <div
              key={index}
              {...a11yOnClick(() => navigate(`/locations/location-detail/${park.abbreviation}?tab=notes`))}
              className='cursor-pointer'
            >
              <ListRow>
                <div className='flex flex-col gap-1'>
                  <h3>{park.name}</h3>
                  <p>{park.address[0].city}</p>
                  <p className='mb-2 text-gray-500 text-sm'>Last updated: Not available</p>
                  <p
                    className='overflow-wrap-anywhere line-clamp-3 max-w-full hyphens-auto break-words'
                    style={{ hyphens: 'auto' }}
                  >
                    {park.parkNotes}
                  </p>
                </div>
              </ListRow>
            </div>
          ))
        )}
      </div>
    </div>
  );
};
