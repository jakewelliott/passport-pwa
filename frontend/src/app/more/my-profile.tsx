import { LoadingPlaceholder } from '@/components/loading-placeholder';
import RoundedButton from '@/components/rounded-button';
import { useUser } from '@/hooks/queries/useUser';
import { dbg } from '@/lib/debug';
import type { UserProfile } from '@/types';
import { useNavigate } from 'react-router-dom';

const AccountInfo = ({ user }: { user: UserProfile }) => {
    return (
        <div>
            <p>Username: {user.username}</p>
            <p>Role: {user.role}</p>
            {/* <p>Account created on: {DateHelper.toStringLong(user.createdAt)}</p> */}
        </div>
    );
};

export default function MyProfileScreen() {
    dbg('RENDER', '/more/my-profile');

    const { data: user, isLoading } = useUser();
    const navigate = useNavigate();

    const handleLogout = () => {
        const popup = window.confirm('Are you sure you want to logout?');
        if (popup) {
            navigate('/logout');
        }
    };

    const handlePasswordChange = () => {
        // TODO: implement password change
    };

    if (isLoading) return <LoadingPlaceholder what='your profile' />;

    return (
        <div className='flex flex-col gap-4'>
            <h2>My Profile</h2>
            {user && <AccountInfo user={user} />}
            <RoundedButton onClick={handlePasswordChange} title='Change Password' />
            <RoundedButton onClick={handleLogout} title='Logout' />
        </div>
    );
}
