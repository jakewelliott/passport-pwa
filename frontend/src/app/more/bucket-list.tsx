import { BucketListItem } from '@/components/tabs/more/bucket-list-item';

export const BucketList = () => {
  return (
    <div className='p-7'>
      <div className='max-w-96 mx-auto m-7'>
        <BucketListItem />
        <BucketListItem />
        <BucketListItem />
        <BucketListItem />
        <BucketListItem />
        <BucketListItem />
        <BucketListItem />
        <BucketListItem />
        <BucketListItem />
        <BucketListItem />
        <BucketListItem />
      </div>
    </div >
  );
};
