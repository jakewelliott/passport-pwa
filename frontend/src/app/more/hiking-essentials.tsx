import { HikingEssentialItem } from '@/components/tabs/more/hiking-essential-item';

export const HikingEssentials = () => {
  return (
    <div className='mx-auto max-w-4xl p-6' lang='en'>
      <h3 className='mb-5 bg-supporting_inactiveblue p-3 text-center text-system_white uppercase'>
        WHAT TO PACK FOR A HIKE
      </h3>

      <div className='flex flex-wrap justify-center gap-6'>
        <HikingEssentialItem
          imageSrc='/icons/hiking/boot.svg'
          altText='Hiking boot'
          title='HIKING BOOTS'
          description='Make sure your boots are broken in before hitting the trails to prevent blisters and sore feet!'
        />

        <HikingEssentialItem
          imageSrc='/icons/hiking/water-bottle.svg'
          altText='Water bottle'
          title='WATER'
          description="You need to drink about 1 liter of water per every 4-5 miles hiked. Don't forget water for your pet!"
        />

        <HikingEssentialItem
          imageSrc='/icons/hiking/map-and-compass.svg'
          altText='Map and compass'
          title='MAP & COMPASS'
          description="Pick up a park map from the park's visitor center or download a digital version at ncparks.gov"
        />

        <HikingEssentialItem
          imageSrc='/icons/hiking/snacks.svg'
          altText='Snacks'
          title='SNACKS'
          description='You can make your own trail mix with dark chocolate, almonds, raisins and unsalted popcorn!'
        />

        <HikingEssentialItem
          imageSrc='/icons/hiking/rain-gear.svg'
          altText='Rain gear'
          title='RAIN GEAR'
          description='When hiking in wet weather, avoid wearing cotton - it traps water and takes a long time to dry.'
        />

        <HikingEssentialItem
          imageSrc='/icons/hiking/flashlight.svg'
          altText='Flashlight'
          title='FLASHLIGHT'
          description='Due to all the trees, the trails will get darker before the rest of the park.'
        />

        <HikingEssentialItem
          imageSrc='/icons/hiking/first-aid-kit.svg'
          altText='First aid kit'
          title='FIRST AID KIT'
          description='Make sure your kit has bandages, antiseptic, antihistamines, and tweezers.'
        />

        <HikingEssentialItem
          imageSrc='/icons/hiking/matches.svg'
          altText='Matches'
          title='MATCHES'
          description='Pack waterproof matches or carry them in a waterproof container.'
        />

        <HikingEssentialItem
          imageSrc='/icons/hiking/knife.svg'
          altText='Knife'
          title='KNIFE'
          description='Make sure your knife or multitool has a locking blade to prevent any injuries.'
        />
      </div>
    </div>
  );
};
