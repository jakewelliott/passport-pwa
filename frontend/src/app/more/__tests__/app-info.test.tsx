import { AppInfo } from '@/app/more/app-info';
import { setupTestEnv } from '@/lib/testing/test-wrapper';
import { describe, expect, it } from 'vitest';

const { render } = setupTestEnv();
describe('AppInfo Component - User Stories', () => {
    it('matches snapshot', () => {
        const { container } = render(<AppInfo />);
        expect(container).toMatchSnapshot();
    });
});
