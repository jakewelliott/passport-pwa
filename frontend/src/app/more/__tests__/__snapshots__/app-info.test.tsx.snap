// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AppInfo Component - User Stories > matches snapshot 1`] = `
<div>
  <div
    class="mx-auto max-w-[500px]"
  >
    <div
      class="my-3.5"
    >
      <div
        class="flex h-2/4 flex-col items-center justify-center"
      >
        <img
          alt="North Carolina Department of Parks and Rec White Logo"
          height="103"
          src="/DPRLogoWhite.svg"
          width="136"
        />
        <h3
          class="text-center"
        >
          North Carolina State Parks
        </h3>
        <p
          class="script h-20"
          style="padding-top: 30px;"
        >
          Passport
        </p>
      </div>
    </div>
    <div
      class="p-4"
    >
      <div
        class="bg-supporting_inactiveblue p-2 text-center text-system_white"
      >
        <h3>
          CONTACT US
        </h3>
      </div>
      <div
        class="space-y-2 bg-supporting_lightblue p-2.5 text-center"
      >
        <div>
          <a
            class="text-supporting_inactiveblue no-underline"
            data-discover="true"
            href="/www.ncparks.gov"
          >
            www.ncparks.gov
          </a>
        </div>
        <div
          class="flex justify-center gap-3"
        >
          <a
            class="flex h-8 w-8 items-center justify-center rounded-full bg-supporting_inactiveblue"
            href="https://www.facebook.com/NorthCarolinaStateParks/"
          >
            <svg
              class="text-supporting_lightblue"
              fill="currentColor"
              height="20"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 320 512"
              width="20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M279.14 288l14.22-92.66h-88.91v-60.13c0-25.35 12.42-50.06 52.24-50.06h40.42V6.26S260.43 0 225.36 0c-73.22 0-121.08 44.38-121.08 124.72v70.62H22.89V288h81.39v224h100.17V288z"
              />
            </svg>
          </a>
          <a
            class="flex h-8 w-8 items-center justify-center rounded-full bg-supporting_inactiveblue"
            href="https://www.instagram.com/ncstateparks/"
          >
            <svg
              class="text-supporting_lightblue"
              fill="currentColor"
              height="20"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 448 512"
              width="20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
              />
            </svg>
          </a>
          <a
            class="flex h-8 w-8 items-center justify-center rounded-full bg-supporting_inactiveblue"
            href="https://www.youtube.com/@parksNC"
          >
            <svg
              class="text-supporting_lightblue"
              fill="currentColor"
              height="20"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 576 512"
              width="20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M549.655 124.083c-6.281-23.65-24.787-42.276-48.284-48.597C458.781 64 288 64 288 64S117.22 64 74.629 75.486c-23.497 6.322-42.003 24.947-48.284 48.597-11.412 42.867-11.412 132.305-11.412 132.305s0 89.438 11.412 132.305c6.281 23.65 24.787 41.5 48.284 47.821C117.22 448 288 448 288 448s170.78 0 213.371-11.486c23.497-6.321 42.003-24.171 48.284-47.821 11.412-42.867 11.412-132.305 11.412-132.305s0-89.438-11.412-132.305zm-317.51 213.508V175.185l142.739 81.205-142.739 81.201z"
              />
            </svg>
          </a>
        </div>
        <p
          class="space-y-1"
        >
          <span>
            NC Division of Parks and Recreation
          </span>
          <br />
          <span>
            Dept. of Natural and Cultural Resources
          </span>
          <br />
          <span>
            1615 Mail Service Center
          </span>
          <br />
          <span>
            Raleigh, NC 27699
          </span>
          <br />
          <span>
            (919) 707-9300
          </span>
        </p>
      </div>
    </div>
    <div
      class="p-4"
    >
      <div
        class="bg-secondary_orange p-2 text-center text-system_white"
      >
        <h3>
          DOWNLOAD PARK MAPS
        </h3>
      </div>
      <div
        class="flex items-center bg-supporting_lightorange p-2.5"
      >
        <div
          class="-ml-8 float-left"
        >
          <svg
            fill="currentColor"
            height="90"
            stroke="currentColor"
            stroke-width="0"
            style="transform: rotate(-25deg);"
            viewBox="0 0 24 24"
            width="90"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M2 5L9 2L15 5L21.303 2.2987C21.5569 2.18992 21.8508 2.30749 21.9596 2.56131C21.9862 2.62355 22 2.69056 22 2.75827V19L15 22L9 19L2.69696 21.7013C2.44314 21.8101 2.14921 21.6925 2.04043 21.4387C2.01375 21.3765 2 21.3094 2 21.2417V5ZM16 19.3955L20 17.6812V5.03308L16 6.74736V19.3955ZM14 19.2639V6.73607L10 4.73607V17.2639L14 19.2639ZM8 17.2526V4.60451L4 6.31879V18.9669L8 17.2526Z"
            />
          </svg>
        </div>
        <p
          class="text-center"
        >
          Save paper and save trees by getting a digital map of the park before you visit!
          <br />
          <br />
          Go to ncparks.gov, pick a park, and download the map or get more information.
        </p>
      </div>
    </div>
    <div
      class="p-4"
    >
      <div
        class="mb-3 flex items-center"
      >
        <img
          alt=""
          class="mr-2 h-[50px] w-[50px]"
          src="/icons/misc/Camping.png"
        />
        <div
          class="flex w-full flex-col justify-center"
        >
          <h4
            class="pb-1 text-center text-main_green"
          >
            MAKE A RESERVATION
          </h4>
          <div
            class="flex flex-wrap items-center justify-between gap-2"
          >
            <a
              class="p-mini"
              href="https://www.ncparks.gov/reservations"
            >
              ncparks.gov/reservations
            </a>
            <p
              class="p-mini"
            >
              1-877-722-6762
            </p>
          </div>
        </div>
      </div>
      <div
        class="mt-3 flex items-center"
      >
        <img
          alt=""
          class="mr-2 h-[50px] w-[50px]"
          src="/icons/misc/PicnicShelter.png"
        />
        <p
          class="p-mini"
        >
          Reserve campsites, picnic shelters and other park facilities online or over the phone.
        </p>
      </div>
    </div>
  </div>
</div>
`;
